HW02/Kate_Fedotova/kate_fedotova_hw_02_t_01.rb

```diff
@@ -0,0 +1,19 @@
+log_file = File.open('task_1.txt'){|file| file.read}
+
+unless File.exist?('task_1.txt')
+    abort 'File is not exist!'
+end
+ 
+if File.open('task_1.txt')
+puts 'File open:' + 'task_1.txt'
+end
+
+file = File.new('task_1.txt')
+
+lines = file.readlines
```

Comment:
you can use one line syntax 
```suggestion
lines = File.readlines('task_1.txt')
```